tests/Browser/BlogManageTest.php

```diff
@@ -0,0 +1,562 @@
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title field is required.')
+                    ->assertSee('The content field is required.');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.5: - Create blog: Content input empty
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogContentInputEmpty()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->type('title', 'new title')
+                    ->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The content field is required.');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.6: - Create blog: Title input empty
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogTitleInputEmpty()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->script("CKEDITOR.instances['blog-content'].setData('content');");
+
+            $browser->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title field is required.');
+        });
+    }
+
+    /**
+     * Edit blog.
+     * case 2.7: Edit blog: Fill all input
+     *
+     * @return void
+     */
+    public function testEditBlogFillAllInput()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->click('#blog-edit-2')
+                    ->waitForLocation('/blog/2/edit')
+                    ->type('title', 'user can see title b edit')
+                    ->script("CKEDITOR.instances['content'].setData('content edit');");
+            $browser->press('Submit')
+                    ->assertPathIs($data['path']['blog'])
+                    ->assertSee('Update blog success!')
+                    ->assertSee('user can see title b edit');
```

Comment:
This case not yet show me the changing of title (since have no info about data before change)

Reply:
I fixed